impeller/entity/entity_unittests.cc

```diff
@@ -1843,27 +1843,16 @@ TEST_P(EntityTest, RuntimeEffectSetsRightSizeWhenUniformIsStruct) {
   auto uniform_data = std::make_shared<std::vector<uint8_t>>();
   uniform_data->resize(sizeof(FragUniforms));
   memcpy(uniform_data->data(), &frag_uniforms, sizeof(FragUniforms));
-  contents->SetUniformData(uniform_data);
-
-  Entity entity;
-  entity.SetContents(contents);
 
-  auto context = GetContentContext();
-  RenderTarget target = context->GetRenderTargetCache()->CreateOffscreen(
-      *context->GetContext(), {1, 1}, 1u);
+  auto buffer_view = RuntimeEffectContents::EmplaceVulkanUniform(
```

Comment:
There was only a single test using the ability to read back the uniforms so I factored that logic into a static function and just made it a unit test.